templates/emails/customer-notification-expiring-subscription.php

```diff
@@ -34,7 +34,12 @@
 		echo wp_kses(
 			sprintf(
 				// translators: %1$s: number of days until expiry, %2$s: date in local format.
-				__( 'Your subscription expires in %1$s days — that’s <strong>%2$s</strong>.', 'woocommerce-subscriptions' ),
+				_n(
+					'Your subscription expires in %1$s day — that’s <strong>%2$s</strong>.',
+					'Your subscription expires in %1$s days — that’s <strong>%2$s</strong>.',
+					(int) $subscription_days_til_event,
+					'woocommerce-subscriptions'
```

Comment:
Hey @peterfabian do you know if it would be possible to also fix this in the email Subject? 

<img width="421" alt="Screenshot 2024-10-29 at 12 12 39 pm" src="https://github.com/user-attachments/assets/0814cd9c-6b71-4655-b7b9-c8ca6caa4539">

The problem with the email subject is that it's editable via the email settings so it can't really be variable.

Reply:
I've submitted a draft PR here with a potential solution to this. 
https://github.com/Automattic/woocommerce-subscriptions-core/pull/711